soda/core/soda/execution/metric/numeric_query_metric.py

```diff
@@ -122,6 +124,23 @@ def set_value(self, value):
             self.value = int(value)
         elif isinstance(value, Decimal):
             self.value = float(value)
+        # Check if numpy is installed
+        elif find_spec("pandas") and find_spec("numpy"):
```

Comment:
I personally would prefer if pandas+numpy specific logic would be in a subclass or something like that instead of mixing it here. We do not exactly have a mechanism for that in the metric classes, I will suggest something if I come up with a something worthwhile considering.